test/packages/other/test_without_service_deployer/manifest.yml

```diff
@@ -0,0 +1,36 @@
+format_version: 3.1.3
+name: test_without_service_deployer
+title: "Package with system tests and no service deployer."
```

Comment:
Looking at the system test config , this package looks like the `log` input package from the integrations repository.

Reply:
This is basically the vanilla package created with `elastic-package create package`, with the system test and some additional fields.